Assets/Scripts/BulletManagement/PlayerEmitter.cs

```diff
@@ -7,7 +7,7 @@ public class PlayerEmitter : EmitterBase
         // For playing bullet animations and audio fx
         public Animator animator;
 
-        protected void Start()
+        protected new void Start()
```

Comment:
We should use the `new` keyword here since we're hiding the `Start()` function from the parent class.

Reply:
Got it, thanks! Fascinating how this is the approach that C# uses to make virtual vs non-virtual distinctions.